CONTRIBUTING.md

```diff
@@ -29,41 +29,58 @@ Pull requests are welcome.
 - You need Python 3.8 installed.
   - This is the minimum supported version of the tool.  Developing with a later version risks introducing type hints such as `list[dict]` that are not compatible with Python 3.8.
 
-## Running without installing
+Initialise Poetry by telling it where Python 3.8 is, e.g.
 
-Use `run.py`.  E.g.
 ```
-python run.py ls "C:\temp\sarif_files"
+# Windows - adjust to the path where you have installed Python 3.8.
+poetry env use "C:\Python38\python.exe"
+# Linux
+poetry env use 3.8
 ```
 
-## Package using `build`
+## Running locally in Poetry virtualenv
 
-Install the [build](https://pypi.org/project/build/) package:
 ```
-python -m pip install --upgrade build
+poetry install
+poetry run python --version
+poetry run sarif --version
```

Comment:
Yes, you're right, this is not strictly necessary, it's because I don't trust tools and am easily confused.  I've made it clearer that this is to check that everything looks OK.